### PR TITLE
Update CHANGELOG and VERSION for v0.2.0 release

### PR DESCRIPTION
**Pull request by ziyouw**
- Set VERSION to v0.2.0
- Create CHANGELOG file for 0.2.0 release.
  List new features and issues fixed

Signed-off-by: Ziyou Wang <ziyouw@vmware.com>

### CODE REVIEW

File: CHANGELOG/CHANGELOG-0.2.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Changelog 0.2
+
+## 0.2.0 - 2022-08-17
+
+### Added
+
+- E2e tests for Theia functions. ([#66](https://github.com/antrea-io/theia/pull/66), [@yanjunz97]) ([#71](https://github.com/antrea-io/theia/pull/71), [@heanlan]) ([#77](https://github.com/antrea-io/theia/pull/77), [@dreamtalen])
```
Comment (dreamtalen) [inline CHANGELOG/CHANGELOG-0.2.md:7]: As suggested by maintainers, we usually don't include tests in the changelog as they are not useful for users.

File: CHANGELOG/CHANGELOG-0.2.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Changelog 0.2
+
+## 0.2.0 - 2022-08-17
+
+### Added
```
Comment (dreamtalen) [inline CHANGELOG/CHANGELOG-0.2.md:5]: Some Grafana enhancement PRs may also be worth listing here. Reference: https://github.com/antrea-io/theia/issues/58

**Review by dreamtalen — COMMENTED** [review]
Thanks Ziyou taking care of this.

File: CHANGELOG/CHANGELOG-0.2.md (hunk 2 above)
Comment (heanlan) [reply CHANGELOG/CHANGELOG-0.2.md:5]: +1. There are three related PRs, can be put in the `Changed` section.

File: CHANGELOG/CHANGELOG-0.2.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+- E2e tests for Theia functions. ([#66](https://github.com/antrea-io/theia/pull/66), [@yanjunz97]) ([#71](https://github.com/antrea-io/theia/pull/71), [@heanlan]) ([#77](https://github.com/antrea-io/theia/pull/77), [@dreamtalen])
+  * Provide e2e tests for Theia components (ClickHouse monitor and Grafana dashboard)
+  * Improve policy recommendation e2e test
+- ClickHouse cluster support and data schema management. ([#55](https://github.com/antrea-io/theia/pull/55), [@yanjunz97]) ([#81](https://github.com/antrea-io/theia/pull/81), [@yanjunz97])
+  * Introduce support for Clickhouse cluster deployment with shards and replicas
+  * Introduce version control to Clickhouse data schema, which detects the data  schema version in initializing stage
```
Comment (yanjunz97) [inline CHANGELOG/CHANGELOG-0.2.md:12]:
```suggestion
  * Introduce version control to ClickHouse data schema, which detects the data schema version in initializing stage
```
Do we need to mention some necessary upgrade would also be applied automatically in initializing stage?

File: CHANGELOG/CHANGELOG-0.2.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+## 0.2.0 - 2022-08-17
+
+### Added
+
+- E2e tests for Theia functions. ([#66](https://github.com/antrea-io/theia/pull/66), [@yanjunz97]) ([#71](https://github.com/antrea-io/theia/pull/71), [@heanlan]) ([#77](https://github.com/antrea-io/theia/pull/77), [@dreamtalen])
+  * Provide e2e tests for Theia components (ClickHouse monitor and Grafana dashboard)
+  * Improve policy recommendation e2e test
+- ClickHouse cluster support and data schema management. ([#55](https://github.com/antrea-io/theia/pull/55), [@yanjunz97]) ([#81](https://github.com/antrea-io/theia/pull/81), [@yanjunz97])
+  * Introduce support for Clickhouse cluster deployment with shards and replicas
```
Comment (yanjunz97) [inline CHANGELOG/CHANGELOG-0.2.md:11]:
```suggestion
  * Introduce support for ClickHouse cluster deployment with shards and replicas
```
Maybe mention we replace the non-cluster mode ClickHouse by ClickHouse cluster?

**Review by yanjunz97 — COMMENTED** [review]
Thanks Ziyou!

Could you also run `make` under `build/charts` to update the version number in Helm docs to resolve the failed test at:  https://github.com/antrea-io/theia/runs/7876956923?check_suite_focus=true

File: CHANGELOG/CHANGELOG-0.2.md (hunk 1 above)
Comment (jianjuns) [reply CHANGELOG/CHANGELOG-0.2.md:7]: Or at least move to "Changed", as it is indeed not about new features.

File: CHANGELOG/CHANGELOG-0.2.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  * Introduce version control to Clickhouse data schema, which detects the data  schema version in initializing stage
+- Theia Command Line enhancement. ([#56](https://github.com/antrea-io/theia/pull/56), [@dreamtalen]) ([#59](https://github.com/antrea-io/theia/pull/59), [@yuntanghsu])
```
Comment (jianjuns) [inline CHANGELOG/CHANGELOG-0.2.md:13]: enhancements

How about "New Theia command line commands"

File: CHANGELOG/CHANGELOG-0.2.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  * Commands for listing and deleting policy recommendation jobs
+  * Commands to retrieve metrics in ClickHouse database
+
+### Changed
+
+- Enhancement to Helm chart based deployment ([#69](https://github.com/antrea-io/theia/pull/69), [@yanjun97]) ([#73](https://github.com/antrea-io/theia/pull/73), [@antoninbas])
```
Comment (jianjuns) [inline CHANGELOG/CHANGELOG-0.2.md:19]: Enhancements

**Comment by ziyouw** [comment]
Thanks for all your comments! I have changed it accordingly, please help to take another look.

File: CHANGELOG/CHANGELOG-0.2.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# Changelog 0.2
+
+## 0.2.0 - 2022-08-17
+
+### Added
+
+- ClickHouse cluster support and data schema management. ([#55](https://github.com/antrea-io/theia/pull/55), [@yanjunz97]) ([#81](https://github.com/antrea-io/theia/pull/81), [@yanjunz97])
+  * Introduce support for ClickHouse cluster deployment with shards and replicas
+  * Introduce version control to ClickHouse data schema, which detects the data schema version and applies necessay upgrade in initializing stage
```
Comment (yanjunz97) [inline CHANGELOG/CHANGELOG-0.2.md:9]:
```suggestion
  * Introduce version control to ClickHouse data schema, which detects the data schema version and applies necessary upgrades in initializing stage
```

File: CHANGELOG/CHANGELOG-0.2.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+  * Introduce version control to ClickHouse data schema, which detects the data schema version and applies necessary upgrades in initializing stage
+- New Theia Command Line commands. ([#56](https://github.com/antrea-io/theia/pull/56), [@dreamtalen]) ([#59](https://github.com/antrea-io/theia/pull/59), [@yuntanghsu])
+  * Commands for listing and deleting policy recommendation jobs
+  * Commands to retrieve metrics in ClickHouse database
+
+### Changed
+
+- Enhancements to Helm chart based deployment ([#69](https://github.com/antrea-io/theia/pull/69), [@yanjun97]) ([#73](https://github.com/antrea-io/theia/pull/73), [@antoninbas])
+- Move Grafana setting files to Persistent Volume ([#84](https://github.com/antrea-io/theia/pull/84), [@yanjun97])
+- E2e tests for Theia functions. ([#66](https://github.com/antrea-io/theia/pull/66), [@yanjunz97]) ([#71](https://github.com/antrea-io/theia/pull/71), [@heanlan]) ([#77](https://github.com/antrea-io/theia/pull/77), [@dreamtalen])
```
Comment (jianjuns) [inline CHANGELOG/CHANGELOG-0.2.md:18]: Move this one to be the last.

File: CHANGELOG/CHANGELOG-0.2.md (hunk 7 above)
Comment (ziyouw) [reply CHANGELOG/CHANGELOG-0.2.md]: Good catch, thanks!

**Review by jianjuns — COMMENTED** [review]
LGTM

File: CHANGELOG/CHANGELOG-0.2.md (hunk 8 above)
Comment (ziyouw) [reply CHANGELOG/CHANGELOG-0.2.md:19]: Done, thanks!

**Review by tnqn — APPROVED** [review]

**Pull request merged**